kornia/color/rgb.py

```diff
@@ -141,14 +142,15 @@ def rgba_to_rgb(image: torch.Tensor) -> torch.Tensor:
 
     # unpack channels
     r, g, b, a = torch.chunk(image, image.shape[-3], dim=-3)
+    r_bg, g_bg, b_bg = bg_rgb
 
     # compute new channels
     a_one = torch.tensor(1.0) - a
```

Comment:
Will this work on GPU?
```suggestion
    a_one = torch.tensor(1.0, device=image.device, dtype=image.dtype) - a
```

Reply:
wouldn't `a_one = 1.0 - a`  work too? I thought torch smartly placed simple ops with scalars to correct device ?

Reply:
@zzyunzhi Can you update this line to either the code suggestion or @cceyda's? Either way  shall work, the previous line looks unsafe to me.